lib/src/integrationTest/java/io/github/taz03/jia/friendships/FriendshipsActionTest.java

```diff
@@ -0,0 +1,24 @@
+package io.github.taz03.jia.friendships;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.github.taz03.jia.TestProxy;
+import io.github.taz03.jia.requests.friendships.FriendshipsActionRequest;
+import io.github.taz03.jia.responses.friendships.FriendshipsActionResponse;
+import org.junit.jupiter.api.Test;
+
+public class FriendshipsActionTest {
+    @Test
+    public void unfollowPkTest() throws Exception{
```

Comment:
first do the follow test and then unfollow the profile, for this test use instagram's official account who's pk is `25025320`

Reply:
do follow inside unfollow or move follow method to top of the code? 
use instagram`s official account pk or [pk from proxy client](https://github.com/Taz03/JIA/pull/37#discussion_r1314276763) ?

Reply:
lets say person A following person B so, A should be us (proxy client pk) and B should be instagram's official account (the pk I mentioned)

Reply:
just noticed A is not in the equation here so yea use instagram's official accounts pk

Reply:
maybe make pk as global variable for 2 tests? No sense to use local